src/infi/clickhouse_orm/query.py

```diff
@@ -342,25 +403,40 @@ def only(self, *field_names):
         qs._fields = field_names
         return qs
 
-    def filter(self, *q, **filter_fields):
+    def _filter_or_exclude(self, *q, **kwargs):
+        reverse = kwargs.pop('reverse', False)
+        prewhere = kwargs.pop('prewhere', False)
+        condition = copy(self._where_q)
```

Comment:
shouldn't this be either _where_q or _prewhere_q, based on whether you're updating the "where" or the "prewhere" clause?

Reply:
Yes, it seems to be a bug, I'll fix it.